content/blog/1.0-announcement/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Open GitOps 1.0 is finally here and why you should care
+date: "2021-11-01T09:00:01.284Z"
+description: "Open GitOps 1.0 brings a mature standard to the DevOps industry."
+author: Dan Garfield and ?
+thumbnail: images/GitOps-1.0.jpg
+---
+
+GitOps is a term coined by Alexis Richardson a few years ago and has grown immensely in popularity. In a nutshell, GitOps offers a new pattern for deploying software that differs from the traditional CD pipeline approach that has dominated the industry for decades. Last year, Amazon, Azure, Codefresh, Github, Redhat, and Weaveworks teamed up to launch the GitOps Working Group under the CNCF. The goal was to formalize all the ideas that have developed around GitOps into a single cohesive set of principles that would be easily digestible and repeatable by any organization.
```

Comment:
```suggestion
GitOps is a term coined by Alexis Richardson a few years ago and has grown immensely in popularity. In a nutshell, GitOps offers a new pattern for deploying software that differs from the traditional CD pipelines approach that has dominated the industry for decades. Last year, Amazon, Azure, Codefresh, Github, Redhat, and Weaveworks teamed up to launch the GitOps Working Group under the CNCF. The goal was to formalize all the ideas that have developed around GitOps into a single cohesive set of principles that would be easily digestible and repeatable by any organization.
```
pipeline → pipelines to align with later usage of pipelines (plural)

Reply:
Here CD pipeline is in agreement with "approach" which I think works better as a singular. Later the plural makes more sense.